src/puppetlabs/trapperkeeper/services/status/check.clj

```diff
@@ -0,0 +1,28 @@
+(ns puppetlabs.trapperkeeper.services.status.check
+  "Shared status check functions."
+  (:require [clojure.java.jdbc :as sql])
+  (:import [java.io File IOException]))
+
+(defn db-up?
+  [db-spec]
+  (try (let [select-42 "SELECT (a - b) AS answer FROM (VALUES ((7 * 7), 7)) AS x(a, b)"
+             [{:keys [answer]}] (sql/query db-spec [select-42])]
+         (= answer 42))
+    (catch Exception _
+      false)))
+
+(defn disk-writable?
+  []
```

Comment:
It seems like this would be better if it took a path argument... how often is the temp directory really the thing that you care about writing to?  And isn't it possible that the temp dir is writable and the real dir isn't?  They may not even be on the same disk partition as one another?

Do you plan on calling this every time a request to the status endpoint is made?  In the case of a load balancer hitting the endpoint very frequently, seems like the extra disk IO (though admittedly tiny) would be undesirable?  If the intent is to only hit it for levels like `:debug`, maybe we should put that in the docstring?